src/normi/_estimators.py

```diff
@@ -94,15 +94,15 @@ class NormalizedMI(BaseEstimator):
     def __init__(
         self,
         *,
-        n_dims: PositiveInt = 1,
+        n_dims: Union[List[PositiveInt], PositiveInt] = 1,
         normalize_method: NormString = 'geometric',
         invariant_measure: InvMeasureString = 'volume',
         k: PositiveInt = 5,
         n_jobs: Int = -1,
         verbose: bool = True,
     ):
         """Initialize NormalizedMI class."""
-        self.n_dims: PositiveInt = n_dims
+        self.n_dims: list[PositiveInt] = n_dims
```

Comment:
```suggestion
        self.n_dims: List[PositiveInt] = n_dims
```

Reply:
`self.n_dims` should have the same type as `n_dims`. But I guess this should become something like `Union[PositiveInt, ArrayLikePostiveInt]`.

Reply:
Good catch. This is fixed.